c/tskit/tables.c

```diff
@@ -7779,18 +7777,9 @@ tsk_ibd_finder_run(tsk_ibd_finder_t *self)
             }
```

Comment:
Do we need to enclose this call to `enqueue_segment` inside `if (seg->right - seg->left > self->min_length)`?

Reply:
I guess it doesn't actually matter in terms of the output, because we filter the segments we record separately. This is reducing the amount of ancestral material track, so should help with performance. Hmm, maybe we should put the check back into ``enqueue_segment``  - I'll update.

Reply:
OK, updated - what do you think @gtsambos?

Reply:
Yeah, this latest version looks good to me. Anything that reduces unnecessary ancestral material is good -- some users may only be interested in the relatively short list of "large" segments